### ISSUE TITLE
Implement StatsZ Zpage

### ISSUE THREAD

**Issue by songy23** [census-instrumentation/opencensus-go#473]
Currently we only have RpcZ. We need one more StatsZ page to display view data for all registered views.

More context here: https://github.com/census-instrumentation/opencensus-java/issues/856.

**Comment by saurvs** [comment]
Hello, I'd like to work on this if nobody else is.

**Comment by songy23** [comment]
@saurvs Generally, contributions are very welcomed! 

I don't think anyone of us are currently working on this? /cc @rakyll @odeke-em @ramonza

**Comment by ramonza** [comment]
@saurvs  Yup, nobody working on this that I'm aware of. Contributions very welcome!

**Comment by songy23** [comment]
@saurvs Hi Saurav, I just noticed that you're a GSOC student, are you interested in officially participating in 2018 OpenCensus GSOC projects? If so, please go through the application here: https://www.linkedin.com/pulse/opencensus-google-summer-code-gopi-palaniappan-pmp/. If that's not the case, you're still very welcome to contribute to OpenCensus :)

/cc @tags07

**Comment by saurvs** [comment]
Hi @songy23. I am interested in GSoC 2018 :) I introduced myself on gitter a few days back and proposed a project not on the official list - building stats and trace library for Rust. @ramonza liked the idea.

I believe working on this issue will help me better understand opencensus's API and write a good proposal.

**Comment by saurvs** [comment]
@songy23 sorry I haven't been able to get to this as I've been busy with other work. I won't be working on it anymore.

**Comment by odeke-em** [comment]
I am coming here from #984 and pasting my opening comment verbatim
>/statz is implemented in OpenCensus-Java https://opencensus.io/zpages/java/#statz but not in OpenCensus-Go. While helping an OpenCensus user over the week, we encountered a need for /statz and they looked at the Java documentation and found it but not for Go which they needed.
